site/en/docs/extensions/mv3/nativeMessaging/index.md

```diff
@@ -31,13 +31,20 @@ An example of the file is as follows:
 
 The native messaging host manifest file must be valid JSON and contain the following fields:
 
-|Name|Description|
-|----|-----------|
-|`name`|Name of the native messaging host. Clients pass this string to [`runtime.connectNative()`][connect-native] or [`runtime.sendNativeMessage()`][send-native-message]. This name can only contain lowercase alphanumeric characters, underscores and dots. The name cannot start or end with a dot, and a dot cannot be followed by another dot.|
-|`description`|Short application description.|
-|`path`|Path to the native messaging host binary. On Linux and macOS the path must be absolute. On Windows it can be relative to the directory in which the manifest file is located. The host process is started with the current directory set to the directory that contains the host binary. For example if this parameter is set to `C:\Application\nm_host.exe` then it will be started with current directory `C:\Application\`.|
-|`type`|Type of the interface used to communicate with the native messaging host. Currently there is only one possible value for this parameter: `stdio`. It indicates that Chrome should use `stdin` and `stdout` to communicate with the host.|
-|`allowed_origins`|List of extensions that should have access to the native messaging host. Wildcards such as `chrome-extension://*/*` are _not_ allowed.|
+`name`
+: Name of the native messaging host. Clients pass this string to [`runtime.connectNative()`][connect-native] or [`runtime.sendNativeMessage()`][send-native-message]. This name can only contain lowercase alphanumeric characters, underscores and dots. The name can't start or end with a dot, and a dot can't be followed by another dot.
+
+`description`
+: Short application description.
+
+`path`
+: Path to the native messaging host binary. On Linux and macOS the path must be absolute. On Windows it can be relative to the directory in which the manifest file is located. The host process is started with the current directory set to the directory that contains the host binary. For example if this parameter is set to `C:\Application\nm_host.exe` then it will be started with current directory `C:\Application\`.
```

Comment:
```suggestion
: Path to the native messaging host binary. On Linux and macOS the path must be absolute. On Windows it can be relative to the directory containing the manifest file. The host process is started with the current directory set to the directory that contains the host binary. For example if this parameter is set to `C:\Application\nm_host.exe` then it will be started with current directory `C:\Application\`.
```